games-strategy/freeciv/freeciv-3.0.9.ebuild

```diff
@@ -0,0 +1,185 @@
+	if ! use dedicated && ! use server ; then
+		ewarn "Disabling server USE flag will make it impossible to start local"
+		ewarn "games, but you will still be able to join multiplayer games."
+	fi
+
+	use system-lua && lua-single_pkg_setup
+}
+
+src_configure() {
+	local myclient=() mydatabase=() myeconfargs=()
+
+	if use auth ; then
+		if ! use mysql && ! use sqlite ; then
+			einfo "No database backend chosen, defaulting"
+			einfo "to mysql!"
+			mydatabase=( mysql )
+		else
+			use mysql && mydatabase+=( mysql )
+			use sqlite && mydatabase+=( sqlite3 )
+		fi
+	else
+		mydatabase=( no )
+	fi
+
+	if use dedicated ; then
+		myclient=( no )
+		myeconfargs+=(
+			--enable-server
+			--enable-freeciv-manual=html
+		)
+	else
+		if use !sdl && use !gtk && ! use qt5 ; then
+			einfo "No client backend given, defaulting to gtk3 client!"
+			myclient=( gtk3 )
+		else
+			use sdl && myclient+=( sdl2 )
+			# Since all gtk3 in gentoo is >= 3.22 we can use the better client
+			use gtk && myclient+=( gtk3.22 )
+			if use qt5 ; then
+				local -x MOCCMD=$(qt5_get_bindir)/moc
+				myclient+=( qt )
+			fi
+		fi
+		myeconfargs+=(
+			$(use_enable server)
+			$(use_enable server freeciv-manual html )
+		)
+	fi
+
+	myeconfargs+=(
+		--enable-aimodules="$(usex aimodules "yes" "no")"
+		--enable-client="${myclient[*]}"
+		--enable-fcdb="${mydatabase[*]}"
+		--enable-fcmp="$(usex modpack "gtk3" "no")"
+		--enable-ipv6
+		# disabling shared libs will break aimodules USE flag
+		--enable-shared
+		--localedir=/usr/share/locale
+		--with-appdatadir="${EPREFIX}"/usr/share/metainfo
+		$(use_enable mapimg)
+		$(use_enable nls)
+		$(use_enable sound sdl-mixer)
+		$(use_enable system-lua sys-lua)
+		$(use_with readline)
+	)
+	econf "${myeconfargs[@]}"
+}
+
+src_install() {
+	default
+
+	if use dedicated ; then
+		rm -rf "${ED}"/usr/share/pixmaps || die
+		rm -f "${ED}"/usr/share/man/man6/freeciv-{client,gtk2,gtk3,modpack,qt,sdl,xaw}* || die
+	else
+		if use server ; then
+			# Create and install the html manual. It can't be done for dedicated
+			# servers, because the 'freeciv-manual' tool is then not built. Also
+			# delete freeciv-manual from the GAMES_BINDIR, because it's useless.
+			# Note: to have it localized, it should be ran from _postinst, or
+			# something like that, but then it's a PITA to avoid orphan files...
+			# freeciv-manual only supports one ruleset argument at a time.
+			for RULESET in alien civ1 civ2 civ2civ3 classic experimental multiplayer sandbox
+			do
+				./tools/freeciv-manual -r ${RULESET} || die
+				docinto html/rulesets/${RULESET}
+				dodoc ${RULESET}*.html
+			done
+		fi
+		if use sdl ; then
+			make_desktop_entry freeciv-sdl "Freeciv (SDL)" freeciv-client
+		else
+			rm -f "${ED}"/usr/share/man/man6/freeciv-sdl* || die
+		fi
+		rm -f "${ED}"/usr/share/man/man6/freeciv-xaw* || die
+	fi
+	find "${ED}" -name "freeciv-manual*" -delete || die
+
+	rm -f "${ED}/usr/$(get_libdir)"/*.a || die
+	find "${ED}" -type f -name "*.la" -delete || die
```

Comment:
You can probably combine these all under a single `find -delete` command, unless there are *.a files outside *libdir* that you want to keep?